application/src/main/java/uk/nhs/hee/tis/revalidation/mapper/DoctorsForDbMapper.java

```diff
@@ -38,4 +39,16 @@ public interface DoctorsForDbMapper {
   default String designatedBodyToConnectionStatus(String designatedBody) {
     return StringUtils.hasLength(designatedBody) ? "Yes" : "No";
   }
+
+  @Mapping(target = "submissionDate", expression = "java("
+      + "uk.nhs.hee.tis.revalidation.util.DateUtil"
+      + ".convertGmcDateToLocalDate(dto.getSubmissionDate()))")
+  @Mapping(target = "dateAdded", expression = "java("
+      + "uk.nhs.hee.tis.revalidation.util.DateUtil.convertGmcDateToLocalDate(dto.getDateAdded()))")
+  @Mapping(target = "underNotice", expression = "java("
+      + "uk.nhs.hee.tis.revalidation.entity.UnderNotice.fromString(dto.getUnderNotice()))")
+  @Mapping(target = "lastUpdatedDate", expression = "java(java.time.LocalDate.now())")
+  @Mapping(target = "existsInGmc", constant = "true")
+  @Mapping(target = "doctorStatus", constant = "NOT_STARTED")
```

Comment:
I think these were shortcuts and reflect things that the Mapper client should decide but there are implications for a number of methods and probably across services.  Do you think this is Tech Improvement to follow?